lib/r10k.rb

```diff
@@ -1,4 +1,11 @@
-module R10K; end
+require 'gettext-setup'
+
+module R10K
+  GettextSetup.initialize(File.absolute_path('../locales', File.dirname(__FILE__)))
+
+  # Attempt to set the R10k error and log message locale
+  FastGettext.locale = ENV["LANG"]
```

Comment:
Does it make sense to have a fallback here? Or what is the behavior if `LANG` isn't set?